slowmovie.py

```diff
@@ -181,6 +187,7 @@ def video_info(file):
 width = epd.width
 height = epd.height
 
+videoInfos = {}
```

Comment:
What's this line for?

Reply:
That's a global variable to keep the video info for each scanned file. It just has to be somewhere before the first `video_info()` call.

Reply:
Ah, I see now! Thanks for the explanation 👍